examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -75,6 +75,12 @@ pub const CIS3_STANDARD_IDENTIFIER: StandardIdentifier<'static> =
 const SUPPORTS_STANDARDS: [StandardIdentifier<'static>; 3] =
     [CIS0_STANDARD_IDENTIFIER, CIS2_STANDARD_IDENTIFIER, CIS3_STANDARD_IDENTIFIER];
 
+/// List of supported entrypoints by the `permit` function (CIS3 standard).
+const SUPPORTS_PERMIT_ENTRYPOINTS: [StandardIdentifier<'static>; 2] = [
+    StandardIdentifier::new_unchecked("updateOperator"),
```

Comment:
You should use [`EntrypointName`](https://docs.rs/concordium-std/latest/concordium_std/struct.EntrypointName.html) instead of `StandardIdentifier`. You'll probably need to mark the `new_unchecked` as a `const` function as well